troposphere/ec2.py

```diff
@@ -916,3 +916,73 @@ class LaunchTemplate(AWSObject):
         'LaunchTemplateData': (LaunchTemplateData, False),
         'LaunchTemplateName': (basestring, False),
     }
+
+
+class FleetLaunchTemplateSpecificationRequest(AWSProperty):
+    props = {
+        'LaunchTemplateId': (basestring, False),
+        'LaunchTemplateName': (basestring, False),
+        'Version': (basestring, False),
+    }
+
+
+class FleetLaunchTemplateOverridesRequest(AWSProperty):
+    props = {
+        'AvailabilityZone': (basestring, False),
+        'InstanceType': (basestring, False),
+        'MaxPrice': (basestring, False),
+        'Priority': (double, False),
+        'SubnetId': (basestring, False),
+        'WeightedCapacity': (double, False),
+    }
+
+
+class FleetLaunchTemplateConfigRequest(AWSProperty):
+    props = {
+        'LaunchTemplateSpecification': (
+            FleetLaunchTemplateSpecificationRequest,
+            False
+        ),
+        'Overrides': (FleetLaunchTemplateOverridesRequest, False),
+    }
+
+
+class OnDemandOptionsRequest(AWSProperty):
+    props = {
+        'AllocationStrategy': (basestring, False),
+    }
+
+
+class SpotOptionsRequest(AWSProperty):
+    props = {
+        'AllocationStrategy': (basestring, False),
+        'InstanceInterruptionBehavior': (basestring, False),
+        'InstancePoolsToUseCount': (integer, False),
+    }
+
+
+class TargetCapacitySpecificationRequest(AWSProperty):
+    props = {
+        'DefaultTargetCapacityType': (basestring, False),
+        'OnDemandTargetCapacity': (integer, False),
+        'SpotTargetCapacity': (integer, False),
+        'TotalTargetCapacity': (integer, False),
+    }
+
+
+class EC2Fleet(AWSObject):
+    resource_type = "AWS::EC2::EC2Fleet"
+    props = {
+        'ExcessCapacityTerminationPolicy': (basestring, False),
+        'LaunchTemplateConfigs': (FleetLaunchTemplateConfigRequest, True),
+        'OnDemandOptions': (OnDemandOptionsRequest, False),
+        'ReplaceUnhealthyInstances': (boolean, False),
+        'SpotOptions': (SpotOptionsRequest, False),
+        'TagSpecifications': [(TagSpecifications), False],
```

Comment:
I believe the syntax here should be `'TagSpecifications': ([TagSpecifications], False),`

Reply:
right as well !